runtime/compilers/rillv1/data/component-template-v1.json

```diff
@@ -145,10 +148,8 @@
         }
       },
       "required": [
-        "col_dimensions",
         "measures",
         "metric_view",
```

Comment:
Nit: we should consolidate on "metrics_view" throughout the codebase, not "metric_view"

Reply:
This is a common change for all components. Will create a separate PR.